patched-fonts/JetBrainsMono/Ligatures/readme.md

```diff
@@ -1,23 +1,22 @@
 [![official JetBrains project](http://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 
-# JetBrains Mono
-A typeface made for developers. \
-More about font features & design can be found on [its page](https://jetbrains.com/mono/). 
+# JetBrains Mono: a typeface made for developers
+![Alt text](images/main-img@2x.png)
```

Comment:
This image and following (new line 77, 83, ...) will not work, I assume?

Reply:
Right. As the link to the image is symlinked to the directory the `.md` file is located, this would mean I would have to duplicate the image directory + the files mentioned in the `.md` for each and every font variation.
I felt like this would add too much data to an already huge repository without a lot of benefit for others.

The only place where I would add it would be the base `readme.md` at the root of `/patched-fonts/JetBrainsMono/`.

Reply:
Actually I was wrong. It's only 7 `.md` files. So I might as well add it. I don't really care for either way.

Reply:
Oh, I would not include/copy the imagery from the JetBrains repository here, I would
* either change the links to full links to the images
* or rather remove the links

```diff
-![Alt text](images/main-img@2x.png)
+![Alt text](https://github.com/JetBrains/JetBrainsMono/blob/master/images/main-img%402x.png)
```

(Normally you would of course not link to `blob/master` but to `blob/v2.251` but that is not yet existing ;)

Reply:
I'll just remove it then.